pkg/segment/structs/evaluationstructs.go

```diff
@@ -408,6 +413,66 @@ func (self *RexExpr) GetNullFields(fieldToValue map[string]utils.CValueEnclosure
 	return findNullFields(self.GetFields(), fieldToValue)
 }
 
+func checkStringInFields(searchStr string, fieldToValue map[string]utils.CValueEnclosure) (bool, error) {
+	for _, v := range fieldToValue {
+		val, err := v.GetString()
```

Comment:
nit: rename `v`. It looks like it's a `CValueEnclosure`, so a name indicating that would be good

Also maybe something like `stringValue` would be a little easier to understand than `val`. What do you think?

Reply:
addressed in latest commit